packages/ui/src/description-list/DescriptionList.tsx

```diff
@@ -28,8 +28,8 @@ function sizeVariant(
 
 const useStyles = makeStyles(
   (theme: SuperDispatchTheme) => ({
-    list: sizeVariant(theme, 2, 1),
-    listSmall: sizeVariant(theme, 1, 0.5),
+    list: sizeVariant(theme, 2, 1.5),
+    listSmall: sizeVariant(theme, 1, 1),
```

Comment:
@umrzoq-toshkentov Have you discussed changes in paddings with Artyom?

Reply:
Yes I have discussed in new design should be changed like that

Reply:
he also asked need to discuss with his team now. Because we have deprecated DescriptionList and DescriptionItem. I think I will revert it. We should update it to DescriptionItem

Reply:
reverted